documentation/src/main/asciidoc/reference/indexing-massindexer.asciidoc

```diff
@@ -217,7 +217,8 @@ you can also use a schema manager to manage schemas manually at the time of your
 <<schema-management-manager>>.
 
 |`purgeAllOnStart(boolean)`
-|`true`
+|Default value depends on <<indexing-massindexer-parameters-drop-and-create-schema,`dropAndCreateSchemaOnStart(boolean)`>>.
+Defaults to `false`, if the mass indexer is configured to drop and create the schema on the start, to `true` otherwise.
```

Comment:
I think it's incorrect to put a comma here in english. Also I think it's "on start"; not "on the start".

```suggestion
|Default value depends on <<indexing-massindexer-parameters-drop-and-create-schema,`dropAndCreateSchemaOnStart(boolean)`>>.
Defaults to `false` if the mass indexer is configured to drop and create the schema on start, to `true` otherwise.
```